data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg

```diff
@@ -174,14 +174,9 @@
             message= _ "Aye, it’s true we do prefer our axes and hammers, but then again, we make swords. We couldna’ claim to be expert weaponsmiths wi’out kenning how to wield the weapons we make."
         [/message]
 
-        [message]
```

Comment:
I suppose one could argue this is redundant, but I find it adds to the 'flavour' of what Hamel is saying. I think it would be a shame to get rid of this entirely without anything to compensate.

Reply:
I don't understand what you mean.

Reply:
I'd prefer to leave this in.